contracts/io/src/lib.rs

```diff
@@ -150,6 +160,7 @@ fn tiles_per_person(players_amount: usize) -> usize {
 }
 
 /// Get random number from BTreeSet
+/// #[cfg(not(test))]
```

Comment:
Why?
```suggestion
```

Reply:
if i delete this it leads to compilation fail